src/webfaf2/templates/_helpers.html

```diff
@@ -140,8 +140,26 @@
 {% if type != 'PYTHON' %}
   <td>{{ frame.symbolsource.path }}</td>
 {% endif %}
-<td>{{ frame.symbolsource.source_path or '-' }}</td>
-<td>{{ frame.symbolsource.line_number or '-' }}</td>
+<td>
+  {%- if frame.symbolsource.source_path %}
+    <abbr title="{{ frame.symbolsource.offset|memory_address }} Build id:{{frame.symbolsource.build_id}}">{{ frame.symbolsource.source_path }}</abbr>
+  {%- elif frame.symbolsource.offset %}
+    {{ frame.symbolsource.offset|memory_address }}
+    {%- if frame.symbolsource.build_id %}
+      Build id:{{frame.symbolsource.build_id}}
```

Comment:
adding <br /> before Build id improves readability a lot and results in less random line breaks.